reference/api/settings.mdx

```diff
@@ -2695,6 +2696,20 @@ This option can be useful when working with large Meilisearch projects. Consider
 **Activating `binaryQuantized` is irreversible.** Once enabled, Meilisearch converts all vectors and discards all vector data that does fit within 1-bit. The only way to recover the vectors' original values is to re-vectorize the whole index in a new embedder.
 </Capsule>
 
+##### `pooling`
+
+Configure how Meilisearch should merge individual tokens into a single embedding.
+
+`pooling` must be one of the following strings:
+
+- `"useModel"`: Meilisearch will fetch the pooling method from the model configuration, default value for new embedders
+- `"forceMean"`: always use mean pooling, default value for embedders created in Meilisearch \<=v1.13
+- `"forceCls"`: always use CLS pooling
```

Comment:
not sure if we can find an authoritative explanation for CLS pooling to add as a link

Reply:
Yeah, I was looking for resources earlier, but most of what I could find was either not authoritative/trustworthy enough, or it was just a section in an article about something else (I even found a LinkedIn post about, I kid you not, BERT/CLS-related drama).

The only thing that could sort of fit is this article on NLP's BERT: https://www.geeksforgeeks.org/explanation-of-bert-model-nlp/, but even then I'm not super convinced.

Reply:
that's super sad :'( 

Without explaining what these means, maybe we could expand on the reason for this parameter to exist?

- In general, users want `useModel`
- The other options were introduced for compatibility with embedders created in Meilisearch <=v1.13 and for compatibility with some remote embedders. 
- The prime example is a composite embedder composed of a local embedder and of a remote embedder that complains about a similarity issue despite using the same model: try changing the `pooling` parameter of the local embedder to the other values and see if it fixes the issue.

Not sure how that fits in the

Reply:
I added a short line explaining the expected usage of the `force` options.

The bit about composite embedders is interesting, but I prefer to have it in the reference for composite embedders.